mpv.rb

```diff
@@ -0,0 +1,68 @@
+class Mpv < Formula
+  # based upon the last version of homebrew-core/Formula/mpv.rb at https://github.com/Homebrew/homebrew-core/commit/41444d526c40b93069b7f0c5414539deb0534179
+  desc "Media player based on MPlayer and mplayer2"
+  homepage "https://mpv.io"
+  url "https://github.com/privatezero/mpv/archive/2019-10-11.tar.gz"
+  version "2019-10-11"
+  sha256 "948eb483718cab755bbd081c9d1c43f101832801ad9db07390e4df4e409cde61"
+  head "https://github.com/mpv-player/mpv.git"
+
+  depends_on "docutils" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python" => :build
+
+  depends_on "ffmpeg"
+  depends_on "jpeg"
+  depends_on "libarchive"
+  depends_on "libass"
+  depends_on "little-cms2"
+  depends_on "lua@5.1"
+
+  depends_on "mujs"
+  depends_on "uchardet"
+  depends_on "vapoursynth"
+  depends_on "youtube-dl"
+
+  def install
+    # LANG is unset by default on macOS and causes issues when calling getlocale
+    # or getdefaultlocale in docutils. Force the default c/posix locale since
+    # that's good enough for building the manpage.
+    ENV["LC_ALL"] = "C"
+
+    args = %W[
+      --prefix=#{prefix}
+      --enable-html-build
+      --enable-javascript
+      --enable-libmpv-shared
+      --enable-lua
+      --enable-libarchive
+      --enable-uchardet
+      --confdir=#{etc}/mpv
+      --datadir=#{pkgshare}
+      --mandir=#{man}
+      --docdir=#{doc}
+      --zshdir=#{zsh_completion}
+    ]
+
+    system "./bootstrap.py"
+    system "python3", "waf", "configure", *args
+    system "python3", "waf", "install"
+
+    system "python3", "TOOLS/osxbundle.py", "build/mpv"
+    prefix.install "build/mpv.app"
+  end
+
+  test do
+    system bin/"mpv", "--ao=null", test_fixtures("test.wav")
+  end
+  
+  def caveats
+    if File.exist?('/usr/local/Caskroom/mpv')
+      <<~EOS
+      Warning: A brew cask installation of MPV was detected, so this version may not be linked.
```

Comment:
If you add a new line at `be linked.`, then it gives a nicer output on 80-char lines.